tests/utils/actions/misc.ts

```diff
@@ -171,6 +171,18 @@ export async function getQuillSelectionText(page: Page) {
   });
 }
 
+export async function getSelectedTextByQuill(page: Page) {
+  return await page.evaluate(() => {
+    const selection = window.getSelection() as Selection;
+    const range = selection.getRangeAt(0);
+    const component = range.startContainer.parentElement?.closest('rich-text');
+    // @ts-ignore
```

Comment:
Prefer to use `@ts-expect-error`

Reply:
OK!

Reply:
> // @ts-ignore and // @ts-expect-error do the exact same thing. They suppress the error the compiler would emit at the line below the comment.
But // @ts-ignore is very generic. It just says: "Compiler, ignore the next line."
But what did you exactly want to suppress?